ext/intl/tests/locale_filter_matches4_icu70.phpt

```diff
@@ -0,0 +1,367 @@
+--TEST--
```

Comment:
According to the existing naming scheme, this should be called locale_filter_matches5.phpt. OTOH, this naming scheme is more self-explaining, so maybe we should use this in the future?